aea/manager/manager.py

```diff
@@ -395,6 +396,7 @@ def add_project(
 
         if not restore:
             project.install_pypi_dependencies()
+            load_all_plugins(is_raising_exception=False)
```

Comment:
what happens when two AEA projects have different version, or version specifier sets, of the same plugin? `pip` should raise an error when the versions are in conflicts. When and how do we handle this?